.travis.yml

```diff
@@ -7,13 +7,17 @@ matrix:
   fast_finish: true
   include:
     - os: linux
+      dist: xenial
```

Comment:
For some reason, travis insists in using an old version of ubuntu, and this version have problems using apt (gpg keys issue)